File: apps/webui/src/branding.ts
/**
 * Product naming (CLAUDE.md §0.1). Mirrors `slas_kernel.branding`; if the product is renamed,
 * change both constants and run the find-and-replace in CLAUDE.md.
 */
export const PRODUCT_NAME = "SW Local Agent Service";
export const SLUG = "slas";
export const AGENT_NAMES = ["Coding Agent", "Validation Agent", "Factory Agent"] as const;
export const PAGES = [
  "Home",
  "Coding",
  "Validation",
  "Factory",
  "Runs",
  "Tickets",
  "Models",
  "Skills",
  "Knowledge",
  "Admin",
] as const;

File: packages/slas-kernel/slas_kernel/branding.py
"""Product naming — the single place the product name lives (CLAUDE.md §0.1).

If the product is renamed, change the constants here and run the one find-replace in
CLAUDE.md. Nothing else in the codebase spells the name out.
"""

from typing import Final

PRODUCT_NAME: Final = "SW Local Agent Service"
SLUG: Final = "slas"
DEFAULT_DATA_ROOT: Final = "/AI/Agent"
AGENT_NAMES: Final[tuple[str, str, str]] = ("coding", "validation", "factory")
